src/hooks/useArrowKeyNavigation.js

```diff
@@ -0,0 +1,90 @@
+import { useRef, useEffect } from 'react';
+
+/**
+ * A react hook to enable arrow key navigation on a component.
+ * @param {*} param0
+ * @returns a useRef, which can be applied to a component
+ */
+
+function handleEnter({ event, currentIndex, activeElement }) {
+  if (currentIndex === -1) { return; }
+
+  activeElement.click();
+  event.preventDefault();
+}
+
+function handleArrowKey({ event, currentIndex, availableElements }) {
+  // If the focus isn't in the container, focus on the first thing
+  if (currentIndex === -1) { availableElements[0].focus(); }
+
+  // Move the focus up or down. Wrap around ends of list.
+  let nextElement;
+  if (event.key === 'ArrowDown') {
+    nextElement = availableElements[(currentIndex + 1) % availableElements.length];
+  }
+
+  if (event.key === 'ArrowUp') {
+    nextElement = currentIndex - 1 < 0
+      ? availableElements[currentIndex - 1 + availableElements.length]
+      : availableElements[currentIndex - 1];
+  }
+  // eslint-disable-next-line no-unused-expressions
+  nextElement && nextElement.focus();
+  event.preventDefault();
+}
+
+/**
+ * Implement arrow key navigation for the given parentNode
+ * @param {object}  options
+ * @param {Event}   options.e          Keydown event
+ * @param {DOMNode} options.parentNode The parent node to operate on. Arrow keys won't navigate outside of this node
+ * @param {String}  options.selectors  Selectors for elements we want to be able to key through
+ */
+function handleEvents({
```

Comment:
I decided it was worth writing and maintaining our own hook for accessibility reasons, as opposede to using one which didn't give us full functionality (no enter key, no clicking, no wrapping around the list)